openff/toolkit/_tests/test_forcefield.py

```diff
@@ -2032,9 +2032,9 @@ def test_some_charges_from_molecule(self, toolkit_registry, force_field):
         omm_system = force_field.create_openmm_system(
             topology, charge_from_molecules=[ethanol], toolkit_registry=toolkit_registry
         )
-        nonbondedForce = [
+        nonbondedForce = next(
             f for f in omm_system.getForces() if type(f) is NonbondedForce
-        ][0]
+        )
```

Comment:
I grant that it's not a huge improvement for something that's just one length long, but there's no downside and it's nice to have this automation capture other cases in which unwrapping the entire iterator is truly wasteful

Currently there's no check that there is only one force, so that's an independent improvement